c2corg_ui/static/js/documentediting.js

```diff
@@ -351,10 +351,16 @@ app.DocumentEditingController.prototype.submitForm = function(isValid) {
     if (this.modelName_ === 'outing') {
       // adapt the Object for what's expected at the API side + format the outing
       this.formatOuting_(data);
+      var userIds = [this.auth_.userData.id];
```

Comment:
this one precisely has to bee discussed with @tsauerwein if we want to change the way that API expects an outing. For now it has to be modified on the UI-side in order to make it acceptable by the API.